book/listings/gobject_properties/4/main.rs

```diff
@@ -0,0 +1,87 @@
+mod custom_button;
+
+use custom_button::CustomButton;
+use glib::BindingFlags;
+use gtk::{glib, Align, Orientation};
+use gtk::{prelude::*, BoxBuilder};
+use gtk::{Application, ApplicationWindowBuilder};
+
+fn main() {
+    // Create a new application
+    let app = Application::new(Some("org.gtk.example"), Default::default())
+        .expect("Initialization failed...");
+    app.connect_activate(on_activate);
+
+    // Run the application
+    app.run();
+}
+
+// When the application is launched…
+fn on_activate(application: &Application) {
+    // … create a new window …
+    let window = ApplicationWindowBuilder::new()
+        .application(application)
+        .title("My GTK App")
+        .build();
+
+    // ANCHOR: buttons
+    // Create the buttons
+    let button_1 = CustomButton::new();
+    let button_2 = CustomButton::new();
+    // ANCHOR_END: buttons
+
+    // ANCHOR: bind_numbers
+    // Assure that "number" of `button_2` is always 1 higher than "number" of `button_1`
+    button_1
+        .bind_property("number", &button_2, "number")
+        // How to transform "number" from `button_1` to "number" of `button_2`
+        .transform_to(|_, value| {
+            let number = value
+                .get::<i32>()
+                .expect("The property needs to be of type `i32`.")
+                .expect("The property needs to be `Some`.");
+            let incremented_number = number + 1;
+            Some(incremented_number.to_value())
+        })
+        // How to transform "number" from `button_2` to "number" of `button_1`
+        .transform_from(|_, value| {
+            let number = value
+                .get::<i32>()
```

Comment:
```suggestion
                .get_some::<i32>()
```

And then you can omit an `unwrap()`. Also above